src/Ast/Builder/CodeBuilder.php

```diff
@@ -195,6 +195,16 @@ public function return(Expr $expr = null, array $attributes = []): Return_
         return new Return_($expr, $attributes);
     }
 
+    public function castToObject(Expr $expr): Expr\Cast
+    {
+        return new Expr\Cast\Object_($expr);
```

Comment:
can you please add uses for Expr\Cast\Object_, Expr\Cast and Expr\Cast\Array_? then if the use with Expr alone is not being used please remove it

Reply:
I added use PhpParser\Node\Expr\Cast but Expr was used before